src/cabanamd.cpp

```diff
@@ -564,4 +565,17 @@ void CabanaMD::check_correctness( int step )
 
 void CabanaMD::print_performance() {}
 
-void CabanaMD::shutdown() { system->destroy(); }
+CabanaMD::~CabanaMD()
+{
+    delete system;
+    delete input;
+
+    if(integrator != nullptr)
+       delete integrator;
+    if(force != nullptr)
+       delete force;
```

Comment:
This gives a warning because I neglected to add a destructor: `virtual ~Force() {}`